dotnet/src/Connectors/Connectors.UnitTests/Memory/Kusto/KustoMemoryStoreTests.cs

```diff
@@ -189,18 +189,17 @@ public async Task ItCanGetMemoryRecordFromCollectionAsync()
         // Arrange
         var expectedMemoryRecord = this.GetRandomMemoryRecord();
         var kustoMemoryEntry = new KustoMemoryRecord(expectedMemoryRecord);
-
         this._cslQueryProviderMock
             .Setup(client => client.ExecuteQueryAsync(
                 DatabaseName,
                 It.Is<string>(s => s.Contains(CollectionName) && s.Contains(expectedMemoryRecord.Key)),
                 It.IsAny<ClientRequestProperties>(),
                 CancellationToken.None))
-            .ReturnsAsync(CollectionToDataReader(new string[][] {
-                new string[] {
+            .ReturnsAsync(CollectionToDataReader(new dynamic[][] {
```

Comment:
Lets keep it serialized, or change it to be an `object[][]`.

Reply:
Changed to `object[][]` as the database is sending `DateTime` object instead of string..